frontend/src/components/Header/StopClock.js

```diff
@@ -90,10 +90,16 @@ class StopClock extends Component<Props, State> {
     const { currentTime } = this.state;
     const { startTime, countdown } = this.props;
     const percentage = getPercRemaining(currentTime, startTime, countdown);
+    const isIE = document.documentMode; // May be deprecated due to browser changes
```

Comment:
Not a fan of this tbh. We should try and make it work for all browsers without the browser specific logic. Is there some sore of compromise that can be done?
If not, there's a module we can use to determine the browser type

Reply:
I tried alot of different things to make it work on all the browsers at the same time. None of them were working properly in all at the same time. I can try it again but i think the end result will be the same. Should I use that module, then?

Reply:
okie dokie if you're confident this is the best solution then let's do it. The module is https://www.npmjs.com/package/detect-browser